rasa/constants.py

```diff
@@ -59,3 +59,5 @@
 
 DEFAULT_SESSION_EXPIRATION_TIME_IN_MINUTES = 60
 DEFAULT_CARRY_OVER_SLOTS_TO_NEW_SESSION = True
+
+DEFAULT_SENDER_ID = "default"
```

Comment:
@ricwo Would it be better to default to generating a random ID when `rasa shell` is called instead of fixing it here as a constant?

The reason for this suggestion is: If you have a persistent tracker store and run `rasa shell`, you would want to start from scratch with a new conversation id and not at whichever state the last session with a default id was in.
(We were also discussing that in the PR that adds such an argument for `rasa interactive`: #5243.)

Reply:
I agree, It was the way I first changed it. @ricwo @erohmensing I am ok with changing it, What is your opinion?

Reply:
@chkoss yes but this is a separate functionality and should probably be controlled with a different command-line argument, something like `--assign-random-conversation-id`

@VictorCoCo your initial implementation assigned a random user ID with every new shell session, and made it impossible to use a deterministic one. being able to define a conversation ID as a command-line argument and having the script choose a random one are two separate issues. whatever the default behaviour, there needs to remain the possibility of a deterministic conversation ID